Jenkinsfile

```diff
@@ -69,7 +71,10 @@ podTemplate(
                     sh('#!/bin/sh -e\n' + 'docker login -u _json_key -p "$(cat /jenkins-docker-builder/credentials.json)" https://eu.gcr.io')
                     sh("docker push $dockerImageName:$dockerImageTag")
                 }
+                stageWithNotify('Send deployment message to Spinnaker','spinnaker-deployment-trigger') {
+                    deployToSpinnaker(dockerImageName: dockerImageName, dockerImageTag: dockerImageTag, kubernetesManifests: "manifests.yaml")
```

Comment:
manifests.yaml needs to exist in this repo, so rename manifest.yaml to manifests.yaml :)

Reply:
oops. Fixed